packages/viewer/src/components/scene-tree/scene-tree.tsx

```diff
@@ -777,7 +777,9 @@ export class SceneTree {
       layout.innerHTML = `
       <vertex-scene-tree-table-column>
         <template>
-          <vertex-scene-tree-table-cell prop:value="{{row.node.name}}" expand-toggle visibility-toggle />
+          <vertex-scene-tree-table-cell prop:value="{{row.node.name}}" expand-toggle visibility-toggle>
+            <div style="width:100%;">{{row.node.name}}</div>
+          </vertex-scene-tree-table-cell>
```

Comment:
I think we can revert this now. However, it should probably be:

```html
<vertex-scene-tree-table-cell prop:value="{{row.node.name}}" expand-toggle visibility-toggle></vertex-scene-tree-table-cell>
```

Depends on the DOCTYPE, but defaults doesn't actually support XHTML style closing tags `</>`.